starter-code/app.js

```diff
@@ -54,5 +54,11 @@ app.locals.title = 'Express - Generated with IronGenerator';
 const index = require('./routes/index');
 app.use('/', index);
 
+const celebritiesRoute = require('./routes/celebrities')
+app.use('/', celebritiesRoute)
```

Comment:
You can also decide to mount the routes on top of a path.

For example:` app.use('/celebrities', celebrityRoutes)`;.

In this way you won't need to prefix every route in the _/routes/celebrities.js_ file with the _/celebrities_ path